tmp/build/Dockerfile

```diff
@@ -1,6 +1,24 @@
-FROM alpine:3.6
+#FROM alpine:3.6
+FROM openshift/origin
 
-RUN adduser -D cluster-samples-operator
+RUN INSTALL_PKGS="dejavu-sans-fonts wget rsync gettext git tar zip unzip shadow-utils" && \
```

Comment:
you actually need all these packages?

Reply:
did not port to new Dockerfile ... I started with them as possible placeholders for future need (dev or debug or support)